### DISCUSSION TITLE
when will 8.0.1 be released

### DISCUSSION THREAD

**Discussion by srxqds** [dotnet/runtime#96472] [General] [answered]
It's been two months after 8.0.0 released

**Comment by huoyaoyuan** [comment, marked as answer]
The PR for updating version number to 8.0.2 (#96422) has been opened, so I think 8.0.1 will be released at patch Tuesday of this month (Jan 9).

**Comment by huoyaoyuan** [comment]
@dotnet/area-infrastructure-libraries Is there any more information about whether a servicing version will be released at each month?

**Reply by ViktorHofer** [reply]
Unfortunately not. Servicing dates aren't public until the release.
